modules/mono/glue/GodotSharp/GodotSharp/Core/Attributes/RPCAttributes.cs

```diff
@@ -3,26 +3,42 @@
 namespace Godot
 {
     [AttributeUsage(AttributeTargets.Method | AttributeTargets.Field | AttributeTargets.Property)]
-    public class RemoteAttribute : Attribute {}
+    public class RemoteAttribute : Attribute
+    {
+    }
```

Comment:
I'm not sure this is an improvement.

Reply:
Looks ok to me. I wouldn't mind keeping the other one if it's possible with editorconfig, but I think it's not.

Reply:
the other option is to have `{ }`. There doesn't seem to be an option for `{}`. I opened https://github.com/dotnet/roslyn/issues/55061.